stdlib/public/core/Set.swift

```diff
@@ -116,14 +116,6 @@
 /// collection wrapper instead of a set. To restore efficient set operations,
 /// create a new set from the result.
 ///
-///     let morePrimes = primes.union([11, 13, 17, 19])
-///
-///     let laterPrimes = morePrimes.filter { $0 > 10 }
-///     // 'laterPrimes' is of type Array<Int>
-///
-///     let laterPrimesSet = Set(morePrimes.filter { $0 > 10 })
-///     // 'laterPrimesSet' is of type Set<Int>
```

Comment:
Why is this deleted?

Reply:
Because it's not relevant anymore.
`Set` defines `filter` to return `Set`.
I fail to find in `Sequence` and `Collection` any good example to replace this snippet with.
There're thing like `prefix` which return `Array` but they all have little sense to apply to sets.

Reply:
I think `map` would be a good example. What do you think?